### PR TITLE
Biruk.api fix

### PR DESCRIPTION
**Pull request by Infinidrix**

### CODE REVIEW

File: Hack/Backend/services/ProbabilityCalculator.js [hunk 1]
```diff
@@ -75,24 +77,28 @@ const setStartDate = () => {
...
+    let probability = await world_probability();
+    return probability;
   }
 
   let startDate = setStartDate();
-  
-  let input_country = iso3to2[input_country_iso];
-  const country = isoConverter[input_country];
+  let iso3_country;
+  if (Object.keys(iso3to2).includes(input_country)){
```
Comment (Khalid-Sultan) [inline Hack/Backend/services/ProbabilityCalculator.js:88]: Would iterating over the keys be necessary? It's a dictionary so I assume iso3to2[input_country] would work right?

File: Hack/Backend/services/ProbabilityCalculator.js [hunk 2]
```diff
@@ -136,24 +142,9 @@ const getCountryStat = async (input_country_iso) => {
...
-      ) {
-        confirmed_sum += parseFloat(item.TotalCases.replace(/,/g, ""));
-        tests_sum += parseFloat(item.TotalTests.replace(/,/g, ""));
-      }
-    });
-  });
-  return confirmed_sum / tests_sum;
+  const latestConfirmed = (await HealthApiParser.getWorldStatistics({query: {criteria: "Confirmed"}})).pop().y;
+  const latestPopuation = (await Population.find({ iso3: "WLD" }).sort("Year")).pop().population;
+  return latestConfirmed/latestPopuation;
```
Comment (Khalid-Sultan) [inline Hack/Backend/services/ProbabilityCalculator.js:147]: Wasn't the formula Confirmed/Tests rather than Confirmed/Population?

Comment (Infinidrix) [reply Hack/Backend/services/ProbabilityCalculator.js:147]: Hmmm, you're right. Is there a way to find the testing stats?

File: Hack/Backend/services/ProbabilityCalculator.js (hunk 1 above)
Comment (Infinidrix) [reply Hack/Backend/services/ProbabilityCalculator.js:88]: Yeah that makes sense

**Pull request merged**